movie_REST_API/movie_REST_API/App/Controllers/InfoMovieViewController.swift

```diff
@@ -75,13 +53,42 @@ final class InfoMovieViewController: UIViewController {
         return button
     }()
 
+    // MARK: - Public Visual Components
+
+    let descpriptionTextView: UITextView = {
+        let text = UITextView()
+        text.font = .systemFont(ofSize: 22)
```

Comment:
```suggestion
        text.font = .systemFont(ofSize: 22)
```
магические числа в константы